src/components/__tests__/CodeMirrorEditor.test.jsx

```diff
@@ -0,0 +1,134 @@
+import CodeMirror from 'codemirror';
+import findLast from 'lodash-es/findLast';
+import last from 'lodash-es/last';
+import React from 'react';
+import ShallowRenderer from 'react-test-renderer/shallow';
+import TestRenderer, {act} from 'react-test-renderer';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
There’s a [fix](https://github.com/benmosher/eslint-plugin-import/pull/794) in the master branch of `eslint-plugin-import` that has yet to be released; once that lands this disable will no longer be necessary (and should itself trigger a lint error for no unnecessary disables)